docs/_includes/generated/security_configuration.html

```diff
@@ -8,6 +8,18 @@
         </tr>
     </thead>
     <tbody>
+        <tr>
+            <td><h5>security.context.factory.class</h5></td>
```

Comment:
the new configuration options are context.factory.class and module.factory.classes.